src/config.js

```diff
@@ -1,22 +1,53 @@
-export default {
-		token: 'secretKey',
-		host: 'localhost',
-		forumUrl: 'https://forum.pasja-informatyki.pl/',
-		port: 3000,
-		mailer: {
-			host: '',
-			port: 587,
-			secure: false,
-			auth: {
-				user: '',
-				pass: ''
-			},
-			tls: {
-				rejectUnauthorized: true
-			}
-		},
-		emailTo: [
-			''
-		]
-}
+import dotenv from 'dotenv';
+import { readFileSync } from 'fs';
+dotenv.config();
+
+export const sslConfig = (() => {
```

Comment:
```suggestion
function getSSLConfig() {
  const PROTOCOL = process.env.PROTOCOL;
  const SSL_KEY_PATH = process.env.SSL_KEY_PATH;
  const SSL_CERT_PATH = process.env.SSL_CERT_PATH;

  let key = "";
  let cert = "";

  if (PROTOCOL === "https") {
    const isSslConfigSpecified = SSL_KEY_PATH && SSL_CERT_PATH;
    
    let sslKeyPath = SSL_KEY_PATH
    let sslCertPath = SSL_CERT_PATH
    
    if (!isSslConfigSpecified) {
      console.warn("SSL config not provided. Self signed certificate is used");

      sslKeyPath = path.resolve(__dirname, "../ssl/develop.key");
      sslCertPath = path.resolve(__dirname, "../ssl/develop.cert");
    }

    key = readFileSync(sslKeyPath, "utf8");
    cert = readFileSync(sslCertPath, "utf8");
  }

  return { key, cert };
}

export const sslConfig = getSSLConfig();
```
Ale jak coś to trzeba sprawdzić, bo z palca pisałem